kayobe-env

```diff
@@ -12,7 +12,7 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-KAYOBE_CONFIG_ROOT=$(dirname $(realpath ${BASH_SOURCE[0]}))
+KAYOBE_CONFIG_ROOT=$(dirname $(realpath ${BASH_SOURCE[0]:-${(%):-%x}}))
```

Comment:
LOL, what sort of voodoo magic is this?

Reply:
Had to sell my soul to the devil to get hold of this spell...